src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -600,3 +600,18 @@ bool Terminal::EndHyperlink() noexcept
     _buffer->SetCurrentAttributes(attr);
     return true;
 }
+
+// Method Description:
+// - Updates the taskbar progress indicator
+// Arguments:
+// - state: indicates the progress state
+// - progress: indicates the progress value
+// Return Value:
+// - true
+bool Terminal::SetTaskbarProgress(const size_t state, const size_t progress) noexcept
+{
+    _taskbarState = state;
+    _taskbarProgress = progress;
+    _pfnTaskbarProgressChanged();

```

Comment:
You should wrap this callback in a `if (_pfnTaskbarProgressChanged)` block - I could imagine the tests or the WPF control not setting the callback, and this causing a crash